backend/src/main/java/WebApi.java

```diff
@@ -7,9 +9,12 @@
     private static WebInterface webInterface = new WebInterfaceImpl();
 
     public static void main(String[] args) {
+        Gson gson = new Gson();
         get("/hello", (req, res) -> "Hello World");
+        get("/dummy/:id", (req, res) -> webInterface.dummy(req.params("id")));
+        get("/digitalPart/:digitalPartID", (req, res) -> webInterface.getDigitalPart(Integer.parseInt(req.params("digitalPartID"))), gson::toJson);
 
-        get("/stl/:stlID", (req, res) -> webInterface.getAllPrintsForStl(req.params("stlID")));
+        System.out.println("SERVER RUNNING!");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)